merino-integration-tests/src/utils/test_tools.rs

```diff
@@ -67,25 +68,39 @@ where
 
     let _tracing_subscriber_guard = tracing::subscriber::set_default(tracing_subscriber);
 
-    // Set up a mock server for Remote Settings to talk to
-    let remote_settings_mock = MockServer::start();
-    remote_settings_mock.mock(|when, then| {
-        when.path("/v1/");
-        then.status(200)
-            .header("Content-Type", "application/json")
-            .json_body(json!({
-                "capabilities": {
-                    "attachments": {
-                        "base_url": remote_settings_mock.base_url()
-                    }
-                }
-            }));
-    });
-    settings.remote_settings.server = remote_settings_mock.base_url();
+    // Set up Remote Settings
+    let reqwest_client = reqwest::Client::new();
+    let bucket_info: serde_json::Value = reqwest_client
+        .post(format!("{}/v1/buckets/", settings.remote_settings.server))
```

Comment:
Unless you run multiple concurrent tests, you could also do a `.put(format!("{}/v1/buckets/{}", settings.remote_settings.server, settings.remote_settings.bucket_id()))`

Reply:
We run multiple concurrent tests.